Anvil/Tools/Actions.pm

```diff
@@ -121,7 +121,7 @@ sub insert_action_node_assume {
 
     # Can we parse a node number from node_name or peer_node_name
     my ($node_number) = $node_name =~ /(\d)/;
-    if (not $number or ($number ne "1" and $number ne "2")) {
+    if (not $node_number or ($node_number ne "1" and $node_number ne "2")) {
```

Comment:
For consistency and code readability (again, heh); We wrap each conditional in it's own (). So here I'd use:

if ((not $node_number) or (($node_number ne "1") and ($node_number ne "2")))
{
   ...

Reply:
It's from the old commits, the latest version has that, I think